src/primitive_esp32.cc

```diff
@@ -305,7 +305,7 @@ PRIMITIVE(total_deep_sleep_time) {
 }
 
 PRIMITIVE(enable_external_wakeup) {
-#ifndef CONFIG_IDF_TARGET_ESP32C3
+#ifdef CONFIG_IDF_TARGET_ESP32
```

Comment:
This doesn't work on the S3?

Reply:
This is supported by ESP32/S2/S3, but not C3.

Reply:
reverted my change.